bin/convert-argv.js

```diff
@@ -86,7 +86,9 @@ module.exports = function(yargs, argv, convertOptions) {
 							registerCompiler(moduleDescriptor[i]);
 							break;
 						} catch(e) {
-							// do nothing
+							if(i === moduleDescriptor.length - 1 && e.message.match(/Cannot find module/)) {
+								throw new Error("Cannot resolve config file. Install appropriate compiler and try again.");
```

Comment:
ok, when it provides multiple options, throwing an error here is wrong, because it only checks the first option.

Maybe throw an error if `i === moduleDescriptor.length` after the loop with all possible packages.